Major updates to instruction flow

AngieRudduck opened the pull request:
# Module: 00
## Lab/Demo: 00

Fixes # .

Changes proposed in this pull request:

-
-
-

AngieRudduck committed:
Screenshots, instructions, order, exercises.

AngieRudduck committed:
Lab 1 instructions update

AngieRudduck committed:
Lab 2 instructions update

AngieRudduck committed:
Updates

AngieRudduck committed:
Lab 5

AngieRudduck committed:
Lab 6 updates

AngieRudduck committed:
Lab 6 updates-2

AngieRudduck committed:
Lab 7&8 changes

AngieRudduck committed:
Lab 8 updates

AngieRudduck committed:
Lab 9 updates

AngieRudduck committed:
Lab 10 updates

AngieRudduck committed:
Tweaks

AngieRudduck committed:
Merge branch 'Main' into 2023-Major-Changes

AngieRudduck committed:
Update 05-create-dax-calculations-in-power-bi-desktop-advanced.md

AngieRudduck reviewed (commented):
Minor updates made based on review.

AngieRudduck committed:
Review changes

AngieRudduck committed:
Zip updates

Mihai-Ac committed:
Fixed headings for comments

removed the numbering from the comments

Mihai-Ac committed:
Update 02-load-data-with-power-query-in-power-bi-desktop.md

Removed numbering from comment lines
fixed some spacing issues

Mihai-Ac committed:
Update 03-configure-data-model-in-power-bi-desktop.md

removed numbering from some comments
fixed indent on some comments

Mihai-Ac committed:
Update 04-create-dax-calculations-in-power-bi-desktop.md

Removed numbering on comments
fixed some headings

Mihai-Ac committed:
Update 05-create-dax-calculations-in-power-bi-desktop-advanced.md

removed numbering from comments 
fixed some headings/spacing

Mihai-Ac commented on Instructions/06-design-report-in-power-bi-desktop.md:
```diff
@@ -9,579 +9,400 @@ lab:
+1. To convert the slicer from a list to a dropdown, at the top-right of the slicer, select the down-arrow, and then select **Dropdown**.
 
-4. In the **Open** window, navigate to the **D:\PL300\Resources** folder.
+    ![Slicer style](Linked_image_Files\06_slicer_style.png)
```
Photo is missing
![image](https://user-images.githubusercontent.com/110993710/209653932-2ec80c5b-29c7-40f9-8c36-83ff269e3c62.png)

Mihai-Ac commented on Instructions/06-design-report-in-power-bi-desktop.md:
```diff
@@ -9,579 +9,400 @@ lab:
+1. To add a chart to the page, in the **Visualizations** pane, select the **Stacked Column Chart** visual type.
 
-	![Picture 26](Linked_image_Files/07-design-report-in-power-bi-desktop_image27.png)
+     ![Picture 53](Linked_image_Files/07-stacked-column-chart.png)
```
Missing image here
![image](https://user-images.githubusercontent.com/110993710/209654295-7e2740db-2d41-45ee-a29c-9b0903a68526.png)

Mihai-Ac commented on Instructions/06-design-report-in-power-bi-desktop.md:
```diff
@@ -9,579 +9,400 @@ lab:
+1. Open a Microsoft Edge browser, then sign-in at **https://app.powerbi.com**.
 
-8. Notice that the bar chart is filtered and highlighted, with the bold portion of the bars representing the filtered months.
+    ![Picture 93](Linked_image_Files/06-my-workspace-new.png)
```
Missing image
![image](https://user-images.githubusercontent.com/110993710/209654640-a796cb64-bb7a-45fd-834e-f2f1e071e3f9.png)

Mihai-Ac committed:
Update 06-design-report-in-power-bi-desktop.md

removed numbering from comments/tips
fixed some headings

Mihai-Ac commented on Instructions/07-design-report-in-power-bi-desktop-enhanced.md:
```diff
@@ -1,488 +1,378 @@
- 	![Picture 22](Linked_image_Files/08-design-report-in-power-bi-desktop-enhanced_image3.png)
-
-7. Leave the Microsoft Edge browser window open.
+     ![Picture 22](Linked_image_Files/07-my-workspace-new.png)
```
Missing picture
![image](https://user-images.githubusercontent.com/110993710/209661050-f8ad49be-7b4e-47f0-a3a7-236874b87e89.png)

Mihai-Ac committed:
Update 07-design-report-in-power-bi-desktop-enhanced.md

removed numbering from comments/tips
fixed some formatting/headings

Mihai-Ac commented on Instructions/08-perform-data-analysis-in-power-bi-desktop.md:
```diff
@@ -9,303 +9,201 @@ lab:
- 	![Picture 4](Linked_image_Files/10-perform-data-analysis-in-power-bi-desktop_image3.png)
-
-1. Leave the Microsoft Edge browser window open.
+     ![Picture 22](Linked_image_Files/07-my-workspace-new.png)
```
Missing photo
![image](https://user-images.githubusercontent.com/110993710/209665013-39a430b3-6f08-4440-a8ac-01552311f749.png)

Mihai-Ac committed:
Update 08-perform-data-analysis-in-power-bi-desktop.md

removed numbering from comments/tips
fixed some spacing/headings

Mihai-Ac commented on Instructions/09-create-power-bi-dashboard.md:
```diff
@@ -9,360 +9,240 @@ lab:
-    ![Picture 40](Linked_image_Files/09-create-power-bi-dashboard_image4.png)
-
-9. Leave the Microsoft Edge browser window open.
+     ![Picture 22](Linked_image_Files/07-my-workspace-new.png)
```
Missing image
![image](https://user-images.githubusercontent.com/110993710/209667821-ea3fb8eb-f675-4cf4-a212-5aea7d317043.png)

Mihai-Ac committed:
Update 09-create-power-bi-dashboard.md

removed numbering from comments/tips
fixed some headings/spacing

Mihai-Ac committed:
Update 10-row-level-security.md

fixed numbering on comments/tips
fixed issue where images would reset numbered lists

Mihai-Ac reviewed (changes requested):
- Checked all labs.

- did not open .pbix files

- Created new commit for each lab with fixes for numbered comments/tips (left comments on each commit).

- Found 4 labs where images were missing (review comments left in the files. They are :

_Instructions/06-design-report-in-power-bi-desktop.md_ : 

**Slicer style](Linked_image_Files\06_slicer_style.png)
Picture 53](Linked_image_Files/07-stacked-column-chart.png)
Picture 93](Linked_image_Files/06-my-workspace-new.png**

For 
_07-design-report-in-power-bi-desktop-enhanced.md_
and
_08-perform-data-analysis-in-power-bi-desktop.md_
and
_09-create-power-bi-dashboard.md_

One image was missing: **Picture 22](Linked_image_Files/07-my-workspace-new.png)**

These images will need to be added, but everything else seems fine.
